Instruction context:
   
 L. 214       310  LOAD_GLOBAL              Socks5Error
                 312  LOAD_GLOBAL              Socks5Error
                 314  LOAD_ATTR                ERR_GENERAL_FAILURE
                 316  CALL_FUNCTION_1       1  '1 positional argument'
                 318  RAISE_VARARGS_1       1  'exception instance'
->             320_0  COME_FROM           298  '298'

../test/15911.pyc -- 
# decompile failed

# file ../test/15911.pyc
# Deparsing stopped due to parse error
